### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by parvez-rashid**
View the screenshot below.

image

### CODE REVIEW

**Comment by Naman-Saxena1** [comment]
Preview link is not working, code looks fine

**Comment by parvez-rashid** [comment]
> Preview link is not working, code looks fine

Will include screenshots for ease. I'l fix the preview link soon.

File: navigation.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            <option value="badges">Navigation</option>
+          </select>
+        </li>
+        <li>
+          <i class="fa-solid fa-house"></i>
+          <a class="options link link-active" href="/navigation.html">home</a>
+        </li>
+        <li>
+          <i class="fa-brands fa-github"></i>
+          <a class="options link" href="/navigation.html">github</a>
```
Comment (Naman-Saxena1) [inline navigation.html]: You should add target="_blank", so that github link opens in new tab.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]

File: navigation.html (hunk 1 above)
Comment (parvez-rashid) [reply navigation.html]: Have updated the code. Please approve PR.

File: navigation.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/navigation.css" />
+    <script
+      src="https://kit.fontawesome.com/43943f2889.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Document</title>
```
Comment (Ankur9669) [inline navigation.html:8]: Use script tag at the bottom

**Review by Ankur9669 — APPROVED** [review]
Code looks great to me!!!!. Nice work

File: navigation.html (hunk 2 above)
Comment (parvez-rashid) [reply navigation.html:8]: Have done it, thanks for the review.

**Review by Naman-Saxena1 — APPROVED** [review]
Looks fine now 👍

**Pull request merged**